examples/places/place-null-categories.yaml

```diff
@@ -0,0 +1,32 @@
+geometry:
+  coordinates:
+  - 0
+  - 0
+  type: Point
+id: overture:places:place:1
+properties:
+  addresses:
+  - freeform: 770 Broadway, Floor 8
+    locality: New York
+  - country: US
+    freeform: '770 Broadway #802'
+    locality: New York
+    region: US-NY
+  brand:
+    name: Example
+    wikidata: Q1000
+  categories: 
+    primary: NULL
```

Comment:
I think here you just want to omit the categories field entirely, rather than explicitly indicate primary as null

Reply:
Fixed